atd-vze/src/views/Locations/Location.js

```diff
@@ -76,6 +77,14 @@ function Location(props) {
     setEditField("");
   };
 
+
+  const downloadAllData = <>
```

Comment:
Could this be using the Button Component? Also the language "Download Global" might not be clear for users. Maybe something like "Download All Crashes" or "Download All-Time Crashes"

![Screen Shot 2020-07-21 at 12 25 06 PM](https://user-images.githubusercontent.com/5697474/88087012-dd10af80-cb4d-11ea-9a61-55a0507571ca.jpg)

Reply:
One particular thing, I didn't like that the text isn't clickable only the floppy disk icon.